frontend/test/ui/pages/desktop/base.py

```diff
@@ -0,0 +1,17 @@
+from pypom import Page
+
+from selenium.webdriver.common.by import By
+
+
+class Base(Page):
+
+    _copter_locator = (By.CLASS_NAME, 'copter')
```

Comment:
Copter is the name of the class in which the firefox is within the ship. It slides up from the bottom at https://testpilot.firefox.com